app/views/submissions/in_review.html.haml

```diff
@@ -1,57 +1,31 @@
-%h1.heading-xlarge
-  Upload processed
+%h1.govuk-heading-xl
+  Review & submit
 
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
 %p
-  Your management information for
-  = @task.id
-  is now ready to be submitted.
-
-.notice
-  %i.icon.icon-important
-    %span.visually-hidden
-      Warning
-  %strong.bold-small
-    Your submission is not complete at this stage. Please review the following:
-
-%p
-  Your uploaded submission file contains:
-
-.data#orders
-  %span.data-item.bold-xsmall
-    Orders
-  %span.data-item.bold-xlarge
-    = @submission.orders_count
-  %span.bold-xsmall
-    rows
-
-.data#invoices
-  %span.data-item.bold-xsmall
-    Invoices
-  %span.data-item.bold-xlarge
-    = @submission.invoices_count
-  %span.bold-xsmall
-    rows
-
-%p
-  Based upon this, your management charge for
-  = task_period_date(@task)
-  has been calculated as:
-
-.data#levy
-  %span.data-item.bold-xlarge
-    = levy_as_string @submission.levy
-
-  %p
-    If everything looks correct please complete this submission.
-
-  %p
-    This submission is not completed until you have clicked this button:
+  Your file has been checked and is ready to submit.
+
+%table.govuk-table
+  %thead.govuk-table__head
+    %tr.govuk-table__row
+      %th.govuk-table__cell Filename
+      %th.govuk-table__cell Invocies
+      %th.govuk-table__cell Orders
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell
+        Uploaded file
```

Comment:
Do we need to add a card to update this with the original filename?

Reply:
Yes I think so?